execution/utility.sh

```diff
@@ -1140,6 +1140,11 @@ function get_cloudformation_stack_output() {
   local stackName="$1"; shift
   local resourceId="$1"; shift
   local attributeType="$1"; shift
+  local mode="$1"; shift
+
+  if [[ -z "${mode}" ]]; then
+    mode="enable"
+  fi
 
```

Comment:
Is this used anymore now that the status checks are gone? Doesn't appear to be

Reply:
Removed